src/staticanalysis/bot/static_analysis_bot/report/base.py

```diff
@@ -135,10 +136,10 @@ def pluralize(word, nb):
             defects='\n'.join(defects),
             analyzers='\n'.join(analyzers),
         )
-        for analyzer, url in patches.items():
+        for patch in patches:
             comment += COMMENT_DIFF_DOWNLOAD.format(
-                analyzer=analyzer,
-                url=url,
+                analyzer=patch.analyzer,
+                url=patch.url or patch.path,
```

Comment:
In which case do we have path and no url? Does it make sense to use the path here?

Reply:
It's only when not running on Taskcluster (for local dev debugging)